src/wp-includes/global-styles-and-settings.php

```diff
@@ -106,7 +107,7 @@ function wp_get_global_stylesheet( $types = array() ) {
 	if ( empty( $types ) && ! $supports_theme_json ) {
 		$types = array( 'variables', 'presets', 'base-layout-styles' );
 	} elseif ( empty( $types ) ) {
-		$types = array( 'variables', 'styles', 'presets' );
```

Comment:
So. Back at Gutenberg I prepared a PR to update the PHPDocs at https://github.com/WordPress/gutenberg/pull/46817 

This is being backported at https://github.com/WordPress/wordpress-develop/pull/3918 though I had to remove the `custom-css` bit from that PR because it had no landed yet. Perhaps it makes more sense to merged the changes at https://github.com/WordPress/wordpress-develop/pull/3918 in this PR and do it altogether?

Reply:
I think it will be faster to merge https://github.com/WordPress/wordpress-develop/pull/3918 (@oandregal 's PR) and rebase here, since it's still a draft.

Reply:
Thanks for the update - good call to go ahead with your PR - this one is still draft as we are still [exploring a change in the custom-css load order](https://github.com/WordPress/gutenberg/pull/47396) which will affect this.